pommerman/forward_model.py

```diff
@@ -348,6 +353,13 @@ def crossing(current, desired):
                 curr_position = agent.position
                 if desired_position != curr_position and \
                       (agent_occupancy[desired_position] > 1 or bomb_occupancy[desired_position] != 0):
+                    if num_agent in kicked_bomb_indexed_by_agent:
```

Comment:
Please put a comment above this saying why we need this. Afaict, it is because this agent is not going to move from its current position and so we are moving any bombs that this agent kicked back to their current position.